### PR TITLE
applications: asset_tracker_v2: Fix bugs related to multicell support in nRF Cloud build

### PR DESCRIPTION
**Pull request by simensrostad**
Fix bugs introduced in https://github.com/nrfconnect/sdk-nrf/pull/5948

### CODE REVIEW

File: applications/asset_tracker_v2/src/cloud/cloud_codec/nrf_cloud/nrf_cloud_codec.c [hunk 1]
```diff
@@ -493,6 +498,9 @@ int cloud_codec_encode_data(struct cloud_codec_data *output,
 		 */
 		int64_t rsrp_ts = modem_dyn_buf->ts;
 
+		/* Adjust RSRP to dBm */
+		modem_dyn_buf->rsrp -= 140;
+
 		len = snprintk(rsrp, sizeof(rsrp), "%d", modem_dyn_buf->rsrp);
 		if ((len < 0) || (len >= sizeof(rsrp))) {
 			LOG_ERR("Cannot convert RSRP value to string, buffer to small");
```
Comment (jayteemo) [inline applications/asset_tracker_v2/src/cloud/cloud_codec/nrf_cloud/nrf_cloud_codec.c:506]: should be "buffer too small" in this and the other log msgs.

**Review by jayteemo — APPROVED** [review]

**Review by plskeggs — APPROVED** [review]
LGTM

File: applications/asset_tracker_v2/src/cloud/cloud_codec/nrf_cloud/nrf_cloud_codec.c (hunk 1 above)
Comment (simensrostad) [reply applications/asset_tracker_v2/src/cloud/cloud_codec/nrf_cloud/nrf_cloud_codec.c:506]: I always confuse "to" and "too" 😢

**Review by jorgenmk — APPROVED** [review]

**Review by jtguggedal — APPROVED** [review]

**Review by jfischer-no — CHANGES_REQUESTED** [review]
>Due to
>#5948 RSRP values sent
>out by the modem module is no longer in dBm format for the nRF Cloud
>build.

It does not make sense to reference url or the number of PR, especially since it includes three commits, please use git commit description style 'commit <12+ chars of sha1>:

```
    Due to commit f65acf3dffb4 ("applications: asset_tracker_v2:
    Add option to convert RSRP/RSRQ to dB"),
    RSRP values sent out by the modem module is no longer
    in dBm format for the nRF Cloud build.
```

**Comment by simensrostad** [comment]
> > Due to
> > #5948 RSRP values sent
> > out by the modem module is no longer in dBm format for the nRF Cloud
> > build.
> 
> It does not make sense to reference url or the number of PR, especially since it includes three commits, please use git commit description style 'commit <12+ chars of sha1>:
> 
> ```
>     Due to commit f65acf3dffb4 ("applications: asset_tracker_v2:
>     Add option to convert RSRP/RSRQ to dB"),
>     RSRP values sent out by the modem module is no longer
>     in dBm format for the nRF Cloud build.
> ```

Thanks! Updated the commit message as per your suggestion.

**Review by jfischer-no — APPROVED** [review]

**Pull request merged**